### PR TITLE
[CSM] Collect and expose AUID

### PR DESCRIPTION
**Pull request by Gui774ume**
### What does this PR do?

This PR exposes the login UID by introducing the `auid` field.

### Motivation

This field can be used to better audit host based process activity.

### Describe how to test/QA your changes

Functional tests were added to test this new feature.

### CODE REVIEW

**Review by cswatt — APPROVED** [review]
approved by docs

**Review by paulcacheux — APPROVED** [review]

File: pkg/security/secl/model/model_unix.go [hunk 1]
```diff
@@ -46,13 +46,14 @@ type Event struct {
...
-	Exit     ExitEvent     `field:"exit" event:"exit"`     // [7.38] [Process] A process was terminated
-	Syscalls SyscallsEvent `field:"-"`
+	Exec          ExecEvent          `field:"exec" event:"exec"`     // [7.27] [Process] A process was executed or forked
+	SetUID        SetuidEvent        `field:"setuid" event:"setuid"` // [7.27] [Process] A process changed its effective uid
+	SetGID        SetgidEvent        `field:"setgid" event:"setgid"` // [7.27] [Process] A process changed its effective gid
+	Capset        CapsetEvent        `field:"capset" event:"capset"` // [7.27] [Process] A process changed its capacity set
+	Signal        SignalEvent        `field:"signal" event:"signal"` // [7.35] [Process] A signal was sent
+	Exit          ExitEvent          `field:"exit" event:"exit"`     // [7.38] [Process] A process was terminated
+	Syscalls      SyscallsEvent      `field:"-"`
+	LoginUIDWrite LoginUIDWriteEvent `field:"-"`
```
Comment (spikat) [inline pkg/security/secl/model/model_unix.go:56]: Are we not willing to serialize the event?

**Review by spikat — APPROVED** [review]
LGTM, I just had a question,
And I'll create a card to not forget about trying to add it too in ebpfless

**Pull request merged**